docs/source/userguide/mesh.rst

```diff
@@ -312,3 +312,296 @@ Meshes from FEniCS
 ------------------
 
 See the `FEniCS documentation <https://fenicsproject.org/olddocs/dolfin/latest/python/demos/built-in-meshes/demo_built-in-meshes.py.html>`_ for more built-in meshes.
+
+
+Meshes from GMSH
+----------------
```

Comment:
This can actually go replace the [GMSH section](https://festim--885.org.readthedocs.build/en/885/userguide/mesh.html#gmsh-example) instead